its/ruling/src/test/resources/expected/python-S2638.json

```diff
@@ -82,6 +85,13 @@
 'project:django-cms-3.7.1/cms/tests/test_permmod.py': [
 665
 ],
+'project:mypy-0.782/mypy/semanal.py': [
+4682,
+4749
```

Comment:
This one looks like a TP but it was hard for me to see why: the parameter `t` got renamed `typ`.
I'm not sure if there is something we could do to improve the reporting here, but it's not ideal (not asking for change here, and not even sure we should create a ticket because I can imagine the cost to benefit would be pretty bad). Just wanted to share this, happy to discuss.